meetings/2021-08-11.md

```diff
@@ -24,42 +24,50 @@ https://meet.jit.si/solid-authorization
 Barath: There is some interesting material on https://lamport.azurewebsites.net/tla/tla.html.
 Someone used it to model an access control system https://arxiv.org/pdf/1806.09848.pdf.
 
-
 ### Effective ACR
 
 Henry: If you have to use the default ACL, the client will have to make 1 request per container to go all the way to the default.
 
 Sarven: You can't pick a random number of depth in container hierarchy. You can have a simple solution where 1 access control is on the default directory and you have 1 level deep.
 
-Henry: I asked how it is meant to be done and this is what the process costs in terms of requests.
+Henry: In the discussion on issue 244 I asked for clarifications on how the effective ACR process was meant to work ([here](https://github.com/solid/authorization-panel/pull/244#discussion_r686050849) and [here](https://github.com/solid/authorization-panel/pull/244#discussion_r686060946)), and then, following Sarven's explanation, I wrote up what the process costs in terms of requests.
 
 Sarven: What is the criteria for fixing it? Is it the number of requests?
 
-Henry: Having a default is not a bad idea. It is a complicated process to get there.
+Henry: Yes, having a minimal number of requests with the default behavior is what is needed. Since having default rules is not a bad idea. It may require a some new features to get us there.
```

Comment:
```suggestion
Henry: Yes, having a minimal number of requests in the default behavior is what is needed. Having default rules is not a bad idea. It may require a some new features to get us there.
```